sapi/cli/php_cli.c

```diff
@@ -317,10 +313,9 @@ static size_t sapi_cli_ub_write(const char *str, size_t str_length) /* {{{ */
 	while (remaining > 0)
 	{
 		ret = sapi_cli_single_write(ptr, remaining);
-		if (!ret) {
-#ifndef PHP_CLI_WIN32_NO_CONSOLE
```

Comment:
The removal of this macro seems wrong.

Reply:
I look `php_handle_aborted_connection` through, it works with `PHP_CLI_WIN32_NO_CONSOLE`.

When at Win32, why not call `php_handle_aborted_connection`?

Reply:
In addition to the usual `php.exe` there is a binary variant `php-win.exe` dedicated to specifically have no console I/O. 

Thanks.